src/Calendar.js

```diff
@@ -86,6 +86,19 @@ class Calendar extends React.Component {
      */
     defaultView: PropTypes.string,
 
+    /**
+     * Days to display in multi-days views.
+     *
+     * The values should indicate days of the week according to Date.prototype.getDay(),
+     * e.g. to show Tue - Fri, the array should be [2, 3, 4, 5].
+     *
+     * When navigating with next and previous, disabled days are skipped
+     *
+     * @type number[]
+     * @default null
+     */
+    enabledDays: PropTypes.arrayOf(PropTypes.number),
```

Comment:
this essentially has to be a contiguous array, since month view can't handle holes in a week. why not make this a range e.g. start/end like we do already in timeGrid